Clarifying confirmation page and notification URL for KP

frederikvig opened the pull request.

brianweet commented on src/Klarna.Payments/README.md:
```diff
@@ -71,15 +71,17 @@ Set the colors and border size for the Klarna widget. The Klarna logo should be
-Notification url is called by Klarna for fraud updates. See further in the documentation for an example implementation. The 'Send product and image URL' checkbox indicates if the product (in cart) page and image URL should be sent to Klarna. When the 'Use attachment' checkbox is checked the developer should send extra information to Klarna. See the Klarna documentation for more explanation: https ...
+In Quicksilver the order confirmation page URL would look something like this: '/en/checkout/order-confirmation'.
+
+[Notification url](https://developers.klarna.com/api/#payments-api__create-a-new-credit-sessionmerchant_urls__notification) is called by Klarna for updates. See further in the documentation for an example implementation or in the [demo site](/demo/Sources/EPiServer.Reference.Commerce.Site/Features/Checkout/Controllers/KlarnaPaymentController.cs#L63) and would be '/klarnaapi/fraud'. The 'Send prod ...
```
`is called by Klarna for updates` -> `is called by Klarna for fraud updates`?

frederikvig commented:
Fixed it now and also updated a KCOV2 use case link to the latest: https://developers.klarna.com/documentation/klarna-checkout/external-payment-methods/
